pyunlocbox/tests/test_functions.py

```diff
@@ -51,8 +51,10 @@ def assert_equivalent(param1, param2):
         assert_equivalent({'y': 3.2}, {'y': lambda: 3.2})
         assert_equivalent({'A': None}, {'A': np.identity(3)})
         A = np.array([[-4, 2, 5], [1, 3, -7], [2, -1, 0]])
+        pinvA = np.linalg.pinv(A)
         assert_equivalent({'A': A}, {'A': A, 'At': A.T})
```

Comment:
Here I need to add the pinvA matrix because the `proj_linalg` cannot compute it from the operator A, but only from the matrix A.